src/Sparrow/Json/LazyStringValue.cs

```diff
@@ -196,15 +196,19 @@ public bool Equals(string other)
             if (_string != null)
                 return string.Equals(_string, other, StringComparison.Ordinal);
 
-            var sizeInBytes = Encodings.Utf8.GetMaxByteCount(other.Length);
 
-            if (_lazyStringTempComparisonBuffer == null || _lazyStringTempComparisonBuffer.Length < other.Length)
-                _lazyStringTempComparisonBuffer = new byte[Bits.PowerOf2(sizeInBytes)];
+            if (_lazyStringTempComparisonBuffer == null || _lazyStringTempComparisonBuffer.Length < other.Length * 5)
+            {
+                var sizeInBytes = Encodings.Utf8.GetMaxByteCount(other.Length);
```

Comment:
Why bother with this entirely? Can't we just specify that the size is *5 and be done with it entirely?

Reply:
Yes, but how much do we grow it afterward. I prefer to have a tighter number, which that calls provide us that.